packages/inter-protocol/src/auction/scheduleMath.js

```diff
@@ -52,14 +75,18 @@ export const computeRoundTiming = (params, baseTime) => {
   const steps = TimeMath.divideRelRel(targetDuration, clockStep);
   const duration = TimeMath.multiplyRelNat(clockStep, steps);
```

Comment:
this value is the same as `actualDuration` so we don't need both. Please use `actualDuration` here and remove the redundant one